action.yml

```diff
@@ -0,0 +1,31 @@
+name: 'Terragrunt Github Action'
+description: 'Guithub Action for Terragrunt.'
+author:  'info@gruntwork.io'
+branding:
+  icon: 'award'
+  color: 'green'
+inputs:
+  tg_version:
+    description: 'Terraform version to install.'
+    default: '1.4.6'
+  tf_version:
+    description: 'Terraform version to install.'
```

Comment:
It looks like `tg_version` and `tf_version` got swapped - should it be `tf_version` with default 1.4.6 and `tg_version` with default 0.46.3?